website/src/data/top-side-menu.yaml

```diff
@@ -0,0 +1,16 @@
+- label: Sign up
+  href: /../getting-started/initial-setup
+- label: Log in
+  href: /../docs/login
```

Comment:
```suggestion
  href: https://docs.commercetools.com/docs/login
```

Reply:
Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1